### DISCUSSION TITLE
HMS series version

### DISCUSSION THREAD

**Discussion by gibzwein** [tbnobody/OpenDTU#243] [Ideas]
Hi,
is it possible to make a version for HMS versions of μinverters? They are working with SUB-1GHz (CC1101 chip).

**Comment by helinhok** [comment]
it's compatible?:  Lora Esp32 Display Oled Wifi Bluetooth 868-915mhz Iot

**Reply by stefan123t** [reply]
It is indeed an CMT2300A and we do not know whether that is using the same Command set as the Gen3 HM-Inverters. Traces and hardware required is discussed in this Ahoy DTU issue https://github.com/lumapu/ahoy/issues/233

**Comment by FrodoVDR** [comment]
https://discord.com/channels/984173303147155506/1082394022359871499
